frontend/app/src/components/inputs/AmountInput.vue

```diff
@@ -26,12 +27,14 @@ const modelValue = defineModel<string>({ required: true });
 const props = withDefaults(defineProps<AmountInputProps>(), {
   hideDetails: false,
   integer: false,
+  rawInput: false,
```

Comment:
when and why do we use this raw Input instead of the RuiTextField one?

Reply:
that's for the amount.
So I want to use the `AmountInput` but I don't want to use the styles for the RuiTextField